test/SmokeTests/OOProcSmokeTests/DotNetIsolated/FaultyOrchestrators.cs

```diff
@@ -0,0 +1,155 @@
+            }
+            
+            // assuming the orchestrator survived the timeout, delete the evidence file and return
+            System.IO.File.Delete(evidenceFile);
+
+            return Task.CompletedTask;
+        }
+
+        [Function("durable_HttpStartOOMOrchestrator")]
+        public static async Task<HttpResponseData> HttpStartOOMOrchestrator(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post")] HttpRequestData req,
+            [DurableClient] DurableTaskClient client,
+            FunctionContext executionContext)
+        {
+            ILogger logger = executionContext.GetLogger("durable_HttpStartOOMOrchestrator");
+
+            // Function input comes from the request content.
+            string instanceId = await client.ScheduleNewOrchestrationInstanceAsync(
+                nameof(OOMOrchestrator));
+
+            logger.LogInformation("Started orchestration with ID = '{instanceId}'.", instanceId);
+
+            // Returns an HTTP 202 response with an instance management payload.
+            // See https://learn.microsoft.com/azure/azure-functions/durable/durable-functions-http-api#start-orchestration
+            return await client.CreateCheckStatusResponseAsync(req, instanceId);
+        }
+
+        [Function("durable_HttpStartProcessExitOrchestrator")]
+        public static async Task<HttpResponseData> HttpStartProcessExitOrchestrator(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post")] HttpRequestData req,
+            [DurableClient] DurableTaskClient client,
+            FunctionContext executionContext)
+        {
+            ILogger logger = executionContext.GetLogger("durable_HttpStartProcessExitOrchestrator");
+
+            // Function input comes from the request content.
+            string instanceId = await client.ScheduleNewOrchestrationInstanceAsync(
+                nameof(ProcessExitOrchestrator));
+
+            logger.LogInformation("Started orchestration with ID = '{instanceId}'.", instanceId);
+
+            // Returns an HTTP 202 response with an instance management payload.
+            // See https://learn.microsoft.com/azure/azure-functions/durable/durable-functions-http-api#start-orchestration
+            return await client.CreateCheckStatusResponseAsync(req, instanceId);
+        }
+
+        [Function("durable_HttpStartTimeoutOrchestrator")]
+        public static async Task<HttpResponseData> HttpStartTimeoutOrchestrator(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post")] HttpRequestData req,
+            [DurableClient] DurableTaskClient client,
+            FunctionContext executionContext)
+        {
+            ILogger logger = executionContext.GetLogger("durable_HttpStartTimeoutOrchestrator");
+
+            // Function input comes from the request content.
+            string instanceId = await client.ScheduleNewOrchestrationInstanceAsync(
+                nameof(TimeoutOrchestrator));
```

Comment:
Rather than replicating the same HTTP function once for each scenario, it might make more sense to have a single HTTP trigger that takes an orchestrator function name as a parameter.

Reply:
I don't feel strongly here, but my personal preference is to specific HTTP endpoints for each orchestrator. When testing locally, it saves me time to just copy-paste the entire URL to trigger the orchestrator. 

But it's no big deal. If you feel strongly, I'll generalize it. Just noting this was a deliberate choice.

Reply:
I don't feel strongly so I'll leave it up to you, but it would enable all the usual benefits of code sharing (easier to add new tests, better consistency b/c only one place to make updates, etc.)